src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -218,7 +216,19 @@ private async Task<Result> EncapsulateFieldAsync(IFieldSymbol field, Document do
             var newDeclaration = newRoot.GetAnnotatedNodes<SyntaxNode>(declarationAnnotation).First();
             field = semanticModel.GetDeclaredSymbol(newDeclaration, cancellationToken) as IFieldSymbol;
 
-            var generatedProperty = GenerateProperty(generatedPropertyName, finalFieldName, originalField.DeclaredAccessibility, originalField, field.ContainingType, new SyntaxAnnotation(), document, cancellationToken);
+            var fieldType = field.GetSymbolType();
+            var fieldNullability = fieldType.GetNullability();
```

Comment:
I totally just realized this was an extension added and the type already has nullability with `field.GetSymbolType()`